src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -52,9 +52,7 @@ def _post(self, request: Request):
         no_cache = request.relay_request_data.get("noCache") or False
```

Comment:
let's move this into `should_use_v3` then and tag by all of those conditions, additionally tag by the return value of `should_use_v3`

Reply:
I didn't move `version` because it's used in the main method and I think it's more readable to add the tag there.